package/MDAnalysis/topology/PDBParser.py

```diff
@@ -515,3 +536,129 @@ def _parse_conect(conect):
+        one PDB ID is given, or a list of strings if multiple PDB IDs are
+        provided.
+
+    Raises
+    ------
+    ValueError
+        For an invalid file format. Supported file formats are under Notes.
+
+    requests.exceptions.HTTPError
+        If an invalid PDB code or file format is specified.
+    
+    Notes
+    -----
+    This function uses the `RCSB File Download Services`_ for directly downloading
+    structure files via https.
+    
+    .. _`RCSB File Download Services`: 
+       https://www.rcsb.org/docs/programmatic-access/file-download-services
+
+    The RCSB currently provides data in 'cif', 'cif.gz', 'bcif', 'bcif.gz', 'xml',
+    'xml.gz', 'pdb', 'pdb.gz', 'pdb1', 'pdb1.gz' file formats and can therefore be
+    downloaded. Not all of these formats can be currently read with MDAnalysis.
+
+    Examples
+    --------
+    Download a single PDB file:
+
+    >>> mda.fetch_pdb("1AKE", file_format="cif")
+    './MDAnalysis_pdbs/1AKE.cif'
+
+    Download multiple PDB files with a progress bar:
+
+    >>> mda.fetch_pdb(["1AKE", "4BWZ"], progressbar=True)
+    ['./MDAnalysis_pdbs/1AKE.pdb.gz', './MDAnalysis_pdbs/4BWZ.pdb.gz']
+
+    Download a single PDB file and converting it to a universe:
+
+    >>> mda.Universe(mda.fetch_pdb("1AKE"), file_format="pdb.gz")
+    <Universe with 3816 atoms>
+
+    Download multiple PDB files and convert each of them into a universe:
+
+    >>> [mda.Universe(mda.fetch_pdb(pdb_id), file_format="pdb.gz") for pdb_id in ("1AKE", "4BWZ")]
+    [<Universe with 3816 atoms>, <Universe with 2824 atoms>]
+
+    
+    .. versionadded:: 2.11.0
+    """
+
+    if not HAS_POOCH:
+        raise ModuleNotFoundError(
+            "pooch is needed as a dependency for fetch_pdb()"
+        )
+    elif file_format not in SUPPORTED_FILE_FORMATS_DOWNLOADER:
+        raise ValueError(
+            f"Invalid file format. Supported file formats are {SUPPORTED_FILE_FORMATS_DOWNLOADER}"
+        )
+    
+    if isinstance(pdb_ids, str):
+        _pdb_ids = (pdb_ids,)
+    else:
+        _pdb_ids = pdb_ids
+
+    if cache_path is None:
+        cache_path = pooch.os_cache("MDAnalysis_pdbs")
+
+    # Have to do this dictionary approach instead of using pooch.retrieve in order
+    # to prevent the hardcoded known_hash warning from showing up.
+    registry_dictionary = {
+        f"{pdb_id}.{file_format}": None for pdb_id in _pdb_ids
+    }
+
+    downloader = pooch.create(
+        path=cache_path,
+        base_url="https://files.wwpdb.org/download/",
+        registry=registry_dictionary,
+    )
+
+    if type(pdb_ids) is str:
+        return str(
+            downloader.fetch(
+                fname=tuple(registry_dictionary.keys())[0],
+                progressbar=progressbar,
+            )
+        )
+    else:
+        return [
+            str(
+                downloader.fetch(fname=file_name, progressbar=progressbar)
+                for file_name in registry_dictionary.keys()
+            )
```

Comment:
Is there a way to simplify the code so that you only `downloader.fetch(fname=...)` once? Along the lines of
```python
paths = [downloader.fetch(fname=file_name, progressbar=progressbar) for file_name in registry_dictionary.keys()]
if type(pdb_ids) is str:
   return paths[0]
return paths
```
or even more concise with the Python ternary operator:
```python
paths = [downloader.fetch(fname=file_name, progressbar=progressbar) for file_name in registry_dictionary.keys()]
return paths if type(pdb_ids) is not str else paths[0]
```

```